docs/04-Bar-Chart.md

```diff
@@ -49,6 +49,7 @@ borderSkipped | `String or Array<String>` | Which edge to skip drawing the borde
 hoverBackgroundColor | `Color or Array<Color>` | Bar background color when hovered
 hoverBorderColor | `Color or Array<Color>` | Bar border color when hovered
 hoverBorderWidth | `Number or Array<Number>` | Border width of bar when hovered
+stack | `String` | For grouping stacked bars
```

Comment:
Ok done!

> About the  stack vs group ,  group  would make sense if this concept could be implemented for every types of chart, which I'm not sure it's possible, right?

I was thinking about folks who extend Chart.js to add other chart types (outside of the core library). If we call this `stack`, they may be tempted to add / use a new property that's better suited to their chart type.

A convoluted example would be a grouped scatter plot (with pie charts instead of dots). `stack` feels out of place, `group` might be ok.